tests/apollo/util/bft.py

```diff
@@ -114,7 +114,7 @@ async def wrapper(*args, **kwargs):
                 nursery.cancel_scope.cancel()
     return wrapper
 
-def with_bft_network(start_replica_cmd, selected_configs=None, num_clients=None, num_ro_replicas=0):
+def with_bft_network(start_replica_cmd, selected_configs=None, num_clients=None, num_ro_replicas=0, rotate_keys=False):
```

Comment:
shouldn't there be changes in the tests too ?

Reply:
yeah, I'm adding them one by one